blockstore/src/test/java/brooklyn/location/blockstore/AbstractVolumeManagerLiveTest.java

```diff
@@ -93,8 +93,12 @@ public void tearDown() throws Exception {
 
     protected static void stripBrooklynProperties(BrooklynProperties props) {
         for (String key : ImmutableSet.copyOf(props.asMapWithStringKeys().keySet())) {
-            if (key.startsWith(BROOKLYN_PROPERTIES_JCLOUDS_PREFIX) && !(key.endsWith("identity") || key.endsWith("credential"))) {
-                props.remove(key);
+            if (key.startsWith(BROOKLYN_PROPERTIES_JCLOUDS_PREFIX) && !(key.endsWith("identity") || key.endsWith("credential") || key.endsWith("loginUser.privateKeyFile"))) {
+                if (!(key.startsWith(BROOKLYN_PROPERTIES_JCLOUDS_PREFIX + "openstack-nova") &&
+                        (key.endsWith("auto-create-floating-ips") || key.endsWith("auto-generate-keypairs") || key.endsWith("keyPair")
+                                || key.endsWith("keystone.credential-type") || key.endsWith("endpoint")))) {
```

Comment:
If these are needed, then perhaps need to reconsider whether filtering for `BROOKLYN_PROPERTIES_JCLOUDS_PREFIX` makes sense.